test/orders/service/OrdersService.spec.ts

```diff
@@ -66,6 +66,42 @@ describe('OrdersService', () => {
         const actualRefund1 = actualOrder.refunds[0];
         expect(actualRefund1.channelRefundId).to.equal(expectedRefund.channelRefundId);
         expect(actualRefund1.supplierRefundId).to.equal(expectedRefund.supplierRefundId);
+        // @ts-ignore
+        expect(2).to.equal(expectedRefund.transactions.length);
+        // @ts-ignore
+        const expectedTransaction1 = actualRefund1.transactions[0];
+        // @ts-ignore
+        const actualTransaction1 = actualRefund1.transactions[0];
+        expect(actualTransaction1.id).to.equal(expectedTransaction1.id);
+        expect(actualTransaction1.amount).to.equal(expectedTransaction1.amount);
+        expect(actualTransaction1.message).to.equal(expectedTransaction1.message);
+        expect(actualTransaction1.status).to.equal(expectedTransaction1.status);
+        // @ts-ignore
+        const expectedTransaction2 = actualRefund1.transactions[1];
+        // @ts-ignore
+        const actualTransaction2 = actualRefund1.transactions[1];
+        expect(actualTransaction2.id).to.equal(expectedTransaction2.id);
+        expect(actualTransaction2.amount).to.equal(expectedTransaction2.amount);
+        expect(actualTransaction2.message).to.equal(expectedTransaction2.message);
+        expect(actualTransaction2.status).to.equal(expectedTransaction2.status);
+        // @ts-ignore
+        expect(2).to.equal(expectedRefund.adjustments.length);
+        // @ts-ignore
+        const expectedAdjustment1 = actualRefund1.adjustments[0];
+        // @ts-ignore
+        const actualAdjustment1 = actualRefund1.adjustments[0];
+        expect(actualAdjustment1.id).to.equal(expectedAdjustment1.id);
+        expect(actualAdjustment1.amount).to.equal(expectedAdjustment1.amount);
+        expect(actualAdjustment1.reason).to.equal(expectedAdjustment1.reason);
+        expect(actualAdjustment1.taxAmount).to.equal(expectedAdjustment1.taxAmount);
+        // @ts-ignore
+        const expectedAdjustment2 = actualRefund1.adjustments[0];
```

Comment:
See my comment above about comparing two of the same objects together. Should just need to change where you are pulling the expected transaction from.